joomla/joomla-cms
# [plg_user_profile] Misc fixes

Pull Request for Issue # .

### Summary of Changes

Added filters and default values to manifest file.
Added filter to ToS field to prevent invalid data from being saved.
Removed `&lang=*` from ToS article URL.
Replaced tooltip with popover for ToS field, to be consistent with other fields.
Some CS (camelCase).

### Testing Instructions

1) Code review. Check that added default values match fallback values in plugin code.
2) Select a ToS article. Check the URL of the article in registration form.
3) When registering, use developer tools in browser to modify ToS input value, e.g.:
`<input id="jform_profile_tos0" name="jform[profile][tos]" value="foobar" required="" aria-required="true" type="radio">`

### Expected result

2) `https://example.com/article-url?tmpl=component`
3) Registration fails because ToS not accepted.

### Actual result

2) `https://example.com/article-url?tmpl=component&lang=*`
3) Registration successful, `"foobar"` written to `#__user_profiles` table as ToS value.

### Documentation Changes Required
No.
